samples/nrf9160/http_update/full_modem_update/README.rst

```diff
@@ -80,8 +80,8 @@ Building and running
 
 .. include:: /includes/build_and_run_ns.txt
 
-The sample is built as a non-secure firmware image for the ``nrf9160dk_nrf9160_ns`` build target.
-Because of this, it automatically includes the :ref:`secure_partition_manager`.
+The sample is built as a firmware image for the ``nrf9160dk_nrf9160_ns`` build target, which separates the firmware between Non-Secure Processing Environment (NSPE) and Secure Processing Environment (SPE).
+Because of this, it automatically includes the :ref:`Trusted Firmware-M (TF-M) <ug_tfm>`.
```

Comment:
@joerchan , should this SPM mention be replaced with TFM? I think it might've been overlooked during the deprecation of SPM a release ago.

Reply:
This sentence feels redundant, because of the .. include:: /includes/build_and_run_ns.txt a few lines above.